salt/states/pkg.py

```diff
@@ -80,10 +80,11 @@
 import os
 import re
 
-# Import salt libs
-import salt.utils
+# Import Salt libs
+import salt.utils  # Can be removed once gen_state_tag is moved
 import salt.utils.pkg
 import salt.utils.platform
+import salt.utils.versions
```

Comment:
Where is this import change used? I don't see any `salt.utils.versions.x` functions anywhere. I do see `from salt.utils.versions import LooseVersion` below the windows gated section, but that's all. Am I missing something?